catalyst_voices/lib/widgets/containers/roles_chooser_panel.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:catalyst_voices/widgets/cards/role_chooser_card.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:flutter/widgets.dart';
+
+/// A panel that displays a series of [RoleChooserCard] widgets for selecting
+/// various account roles (e.g., voter, proposer, drep).
+class RolesChooserPanel extends StatelessWidget {
+  /// A map where keys are [AccountRole] enums and values are booleans
+  /// representing whether the corresponding role is selected.
+  final Map<AccountRole, bool> value;
```

Comment:
would simple `List` or `Set` be sufficient?

then you could say.

```dart
value: value.contains(role),
``` 

`final List<AccountRole> selected` seems more descriptive

Reply:
Makes sense 👍